src/webui/btjson.cpp

```diff
@@ -800,6 +801,7 @@ QVariantMap toMap(BitTorrent::TorrentHandle *const torrent)
     ret[KEY_TORRENT_NUM_INCOMPLETE] = torrent->totalLeechersCount();
     const qreal ratio = torrent->realRatio();
     ret[KEY_TORRENT_RATIO] = (ratio > BitTorrent::TorrentHandle::MAX_RATIO) ? -1 : ratio;
+    ret[KEY_TORRENT_STATUS] = torrent->state().toString();
```

Comment:
>Just committed a workaround for this.

No good, the same value is still sent over the wire twice.
you should do
```js
// dynamicTable.js
            // status
            this.columns['status'].updateTd = function (td, row) {
                var status = this.getRowValue(row);
                if (!status) return;  // add

// client.js
                    if (response['torrents']) {
                        for (var key in response['torrents']) {
                            response['torrents'][key]['status'] = response['torrents'][key]['state']  // add
```
and drop the changes in btjson.cpp.